tests/statistics_test.sh

```diff
@@ -114,20 +100,20 @@ function min_value_Test
   assertEquals "($LINENO)" "$min" "$pre_min"
 }
 
-function sec_to_formatted_date_Test
-{
-  formatted_time=$(sec_to_formatted_date "$pre_total_sec")
-  assertEquals "($LINENO)" "$formatted_time" "$pre_formated_sec"
+# function test_sec_to_formatted_date()
+# {
+#   formatted_time=$(sec_to_formatted_date "$pre_total_sec")
+#   assertEquals "($LINENO)" "$formatted_time" "$pre_formated_sec"
 
-  formatted_time=$(sec_to_formatted_date "")
-  assertEquals "($LINENO)" "$formatted_time" "00:00:00"
-}
+#   formatted_time=$(sec_to_formatted_date "")
+#   assertEquals "($LINENO)" "$formatted_time" "00:00:00"
+# }
```

Comment:
Why?

Reply:
The function `sec_to_formatted_date` was removed from the source code when kw pomodoro was first introduced in #279. But then the testing of that function was brought back by me in a commit after that in #283. 
When I created the PR, `kw pomodoro` hadn't been introduced yet, so my code didn't have this modification.
So, in conclusion, I guess those commented lines can be deleted.

Reply:
Similarly to the other problem, this function was not declared in the `suite` of this test. Will delete it in the next update to this PR.